routes/web.php

```diff
@@ -43,9 +43,7 @@
 
         Route::resource('orders.items', 'OrderItemsController', ['only' => ['update']]);
 
-        Route::get('/', function () {
-            return ujs_redirect(route('admin.store.orders.index'));
-        });
+        Route::get('/', '\App\Http\Controllers\RedirectController')->name('redirect.admin.store.orders.index');
```

Comment:
yeaaaaaah so I guess I just forgot to clear the route cache from when I first added it. I brought this upon myself, didn't I

fixed now :p